src/graph/utils/pm/nested_matcher.cpp

```diff
@@ -200,12 +200,14 @@ bool node_inputs_matcher_t::match_commutative_inputs() {
 
     for (size_t node_input_offset = 0; node_input_offset < node_inputs_.size();
             ++node_input_offset) {
+        if (!has_commutative_inputs(get_op(), {node_input_offset})) continue;
```

Comment:
maybe it's better to change the name of this function to is_commutative_input(), as it's verifying the single input port now.

Reply:
Updated